src/modality-specific-files/task-events.md

```diff
@@ -268,7 +268,11 @@ in the accompanying JSON sidecar as follows (based on the example of the previou
         "SoftwareName": "Psychtoolbox",
         "SoftwareRRID": "SCR_002881",
         "SoftwareVersion": "3.0.14",
-        "Code": "doi:10.5281/zenodo.3361717"
+        "Code": "doi:10.5281/zenodo.3361717",
+        "ScreenDistance": 180,
```

Comment:
1.80 of meters here is very sensible

Reply:
+1 on using SI base units (meters) following Unit appendix (as @robertoostenveld also noted)